package.json

```diff
@@ -20,15 +20,15 @@
   },
   "devDependencies": {
     "rimraf": "^3.0.0",
-    "@types/jest": "^24.0.24",
+    "@types/jest": "^25.1.5",
     "coveralls": "^3.0.9",
     "jest": "^24.9.0",
     "prettier": "^1.19.1",
     "ts-jest": "^24.2.0",
     "typescript": "^3.7.4"
   },
   "peerDependencies": {
-    "jest": "^24.0.0"
+    "jest": "^25.1.5"
```

Comment:
shouldn't force users to update to jest 25 here if it isn't needed right?

I think this line should be     "jest": "^24.0.0 || ^25.0.0"

Reply:
got it